packages/agreement-process/src/services/readmodel/readModelService.ts

```diff
@@ -551,14 +557,10 @@ export function readModelServiceBuilder(
         },
         {
           $match: {
-            "eservices.data.name": {
-              $regex: new RegExp(
-                ReadModelRepository.escapeRegExp(eserviceName || ""),
-                "i"
-              ),
-            },
-            consumerFilter,
-            producerFilter,
```

Comment:
This was causing the bug: filters were added here without object spread so they were not working properly